src/plugins/imagery/components/Compass/CompassRose.vue

```diff
@@ -21,130 +21,134 @@
  *****************************************************************************/
 
 <template>
-<div
-    class="c-direction-rose"
-    @click="toggleLockCompass"
+<div class="w-direction-rose"
```

Comment:
Super duper nitpicky but can we just get the class attribute on a new line to be consistent with the other attributes in the template?

Reply:
Fixed.